src/main/components/ViewTest.jsx

```diff
@@ -0,0 +1,258 @@
+import React, { useState, useEffect } from 'react';
+
+import JsonTree from '../lib/treeView/jsonTree';
+import '../lib/treeView/jsonTree.css';
+
+const ViewTest = (props) => {
+  const [setLoading] = useState(false);
+  // const [data, setData] = useState({});
+  const data = [
```

Comment:
Let’s pull this out into some kind of `constants` or `data` file and import it

Reply:
done